packages/@aws-cdk/aws-codebuild/lib/project-logs.ts

```diff
@@ -0,0 +1,69 @@
+import * as logs from '@aws-cdk/aws-logs';
+import * as s3 from '@aws-cdk/aws-s3';
+
+/**
+ * Information about logs built to an S3 bucket for a build project.
+ */
+export interface S3LogsConfig {
+  /**
+   * Encrypt the S3 build log output
+   * @default true
```

Comment:
Make sure to leave an empty line before the `@default` line:

```suggestion
   * Encrypt the S3 build log output
   *
   * @default true
```

Same comment for the remaining places in this PR.